src/commonMain/kotlin/fr/acinq/lightning/wire/MessageOnion.kt

```diff
@@ -7,17 +7,15 @@ import fr.acinq.bitcoin.io.ByteArrayOutput
 import fr.acinq.bitcoin.io.Input
 import fr.acinq.bitcoin.io.Output
 import fr.acinq.lightning.crypto.RouteBlinding
-import kotlinx.serialization.Contextual
-import kotlinx.serialization.Serializable
 
-@Serializable
+
 sealed class OnionMessagePayloadTlv : Tlv {
     /**
      * Onion messages may provide a reply path, allowing the recipient to send a message back to the original sender.
      * The reply path uses route blinding, which ensures that the sender doesn't leak its identity to the recipient.
      */
-    @Serializable
-    data class ReplyPath(@Contextual val blindedRoute: RouteBlinding.BlindedRoute) : OnionMessagePayloadTlv() {
+
```

Comment:
nit: extra lines between documentation and class (same below)

Reply:
Done in c5365f4.